utils/issue-format-bot/lib/parse.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 AJ Jordan
+//
+// This file is part of the HTTPS Everywhere issue management bot.
+//
+// The HTTPS Everywhere issue management bot is free software: you can
+// redistribute it and/or modify it under the terms of the GNU Affero
+// General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// The HTTPS Everywhere issue management bot is distributed in the
+// hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+// the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+// PURPOSE.  See the GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with the HTTPS Everywhere issue management bot.  If
+// not, see <http://www.gnu.org/licenses/>.
+
+// Takes in an issue body and parses it into key-value pairs, normalizes it, and returns an Object
+
+'use strict';
+
+const strip = require('strip-markdown'),
+      remark = require('remark'),
+      processor = remark().use(strip),
+      _ = require('lodash'),
+      domainFromPartialUrl = require('domain-from-partial-url'),
+      Entities = require('html-entities').AllHtmlEntities,
+      decode = (new Entities()).decode;
+
+// XXX should this be moved to the validator module?
```

Comment:
I think it makes sense to keep them separate, as you have here.  One is for parsing, one is for validating.  They do two separate things.